server/core/services.py

```
@@ -88,6 +89,53 @@ def logout_user(request):
+            
+            if type == "event":
+                querySet = querySet.filter(event__isnull=False)
+
+            if type == "club":
+                querySet = querySet.filter(club__isnull=False)
+
+            if type == "mics":
+                querySet = querySet.filter(mics__isnull=False)
+            
+            # if no posts fit the filter, return all posts
+            results = None if querySet == None else querySet[offset: offset+limit]
```

Ruff (0.8.2)

Comparison to `None` should be `cond is None`

Replace with `cond is None`

(E711)